example/lib/theme/app_color.dart

```diff
@@ -0,0 +1,249 @@
+        toolbar = _KevinColors.extraLightBlue,
+        primaryText = _KevinColors.black,
+        secondaryText = _KevinColors.gray1,
+        surface = _KevinColors.white,
+        onSurface = _KevinColors.black,
+        error = _KevinColors.warningRed,
+        onError = _KevinColors.white,
+        inputUnfocusedBorder = _KevinColors.gray5,
+        bottomSheet = const BottomSheet.light();
+
+  const AppColor.dark()
+      : primary = _KevinColors.blue,
+        primaryVariant = _KevinColors.darkBlue,
+        onPrimary = _KevinColors.white,
+        secondary = _KevinColors.blue,
+        secondaryVariant = _KevinColors.darkBlue,
+        onSecondary = _KevinColors.white,
+        text = _KevinColors.white,
+        controlHighlight = _KevinColors.blue,
+        statusBar = Colors.transparent,
+        navigationBar = _KevinColors.extraLightBlue,
+        primaryBackground = _KevinColors.black,
+        onBackground = _KevinColors.white,
+        secondaryBackground = _KevinColors.gray0,
+        selectedOnSecondary = _KevinColors.blue,
+        rippleOnPrimary = _KevinColors.blue,
+        toolbar = _KevinColors.black,
+        primaryText = _KevinColors.white,
+        secondaryText = _KevinColors.gray1,
+        surface = _KevinColors.gray0,
+        onSurface = _KevinColors.white,
+        error = _KevinColors.warningRed,
+        onError = _KevinColors.white,
+        inputUnfocusedBorder = _KevinColors.black30,
+        bottomSheet = const BottomSheet.light();
+
+  const AppColor({
+    required this.primary,
+    required this.primaryVariant,
+    required this.onPrimary,
+    required this.secondary,
+    required this.secondaryVariant,
+    required this.onSecondary,
+    required this.text,
+    required this.controlHighlight,
+    required this.statusBar,
+    required this.navigationBar,
+    required this.primaryBackground,
+    required this.onBackground,
+    required this.secondaryBackground,
+    required this.selectedOnSecondary,
+    required this.rippleOnPrimary,
+    required this.toolbar,
+    required this.primaryText,
+    required this.secondaryText,
+    required this.surface,
+    required this.onSurface,
+    required this.error,
+    required this.onError,
+    required this.inputUnfocusedBorder,
+    required this.bottomSheet,
+  });
+
+  @override
+  List<Object?> get props => [
+        primary,
+        primaryVariant,
+        onPrimary,
+        secondary,
+        secondaryVariant,
+        onSecondary,
+        text,
+        controlHighlight,
+        statusBar,
+        navigationBar,
+        primaryBackground,
+        onBackground,
+        secondaryBackground,
+        selectedOnSecondary,
+        rippleOnPrimary,
+        toolbar,
+        primaryText,
+        secondaryText,
+        surface,
+        onSurface,
+        error,
+        onError,
+        inputUnfocusedBorder,
+        bottomSheet,
+      ];
+}
+
+class BottomSheet extends Equatable {
```

Comment:
I would move out this class to separate file

Reply:
Reworked in https://github.com/getkevin/kevin-flutter/pull/20/commits/205dce361bc77742a6c6fc88391c686ae645f87a
Renamed to `AppColorBottomSheet` and moved to separate file.